plugin/ocgrpc/grpc_test.go

```diff
@@ -71,21 +69,21 @@ func TestNewClientStatsHandler(t *testing.T) {
 	view.Unsubscribe(ClientErrorCountView)
 }
 
-func TestNewServerStatsHandler(t *testing.T) {
+func TestServerHandler(t *testing.T) {
 	ctx := context.Background()
-
-	handler := NewServerStatsHandler()
-
 	te := &traceExporter{}
 	trace.RegisterExporter(te)
 	if err := ServerRequestCountView.Subscribe(); err != nil {
 		t.Fatal(err)
 	}
 
+	// Ensure we start tracing
```

Comment:
```
// Ensure we start tracing.
```

End with full stop. In Go, block comments are proper English sentences, side comments can be non-proper sentences. For example:

```
ctx = trace.WithSpan(ctx, span) // span in the context
```